app/src/main/java/org/groundplatform/android/MainActivity.kt

```diff
@@ -75,6 +76,8 @@ class MainActivity : AbstractActivity() {
       }
     }
 
+    handleDeepLinkIntent(intent)
```

Comment:
The way this is written implies it will always handle the deep link intent. To make clear what's happening, let's check here: `if (intent.data) { handleDeepLinkIntent(intent.data) }`